### ISSUE TITLE
Add inputs to the cube cleaner.

### ISSUE THREAD

**Issue by ZombiePwner1** [TeamSpen210/BEE2-items#1214]
It would be nice to have some inputs instead of it being automatic.

**Comment by Luke18033** [comment]
I thought it already had this.

**Comment by Luke18033** [comment]
Which should be done first, this or the blue/orange gel variants of the cube cleaner?

**Comment by Endermage77** [comment]
In terms of functionality, the ability to activate/deactivate the cleaner would probably be immediately handier for map-making.
The gel-spritzer can probably wait a bit.

**Comment by Luke18033** [comment]
My idea for this was to cover it with two sliding glass panes when the cleaner is disabled. The static on version would have no glass, like the current item. The static off version would be covered by a single pane of glass, although doing that would be kind of pointless - except in Overgrown, where it would be a destroyed version of the non-static version.

**Comment by Endermage77** [comment]
Nice. Certainly a lot better than my idea, which was to simply turn off the sprinkler when disabled.
